syft_tensorflow/tensor/keras_layer.py

```diff
@@ -0,0 +1,104 @@
+import weakref
+
+import tensorflow as tf
+
+import syft
+from syft.generic.frameworks.hook import hook_args
+from syft.generic.object import AbstractObject
+from syft.generic.pointers.object_pointer import ObjectPointer
+from syft.workers.base import BaseWorker
+
+from syft_tensorflow.tensor import KerasObject
+
+from syft.exceptions import PureFrameworkTensorFoundError
+
+
+class KerasLayer(KerasObject):
+    """Add methods to this keras layer object to have them added to every 
+    tf.keras.layers.Layer object.
+
+    This Keras object is simply a more convenient way to add custom functions to
+    all TensorFlow Keras object types. When you add a function to this keras object,
+    it will be added to EVERY native Keras layer type (i.e. tf.keras.layers.Dense)
+    automatically by the TensorFlowHook.
+
+    Note: all methods from AbstractObject will also be included because this
+    object extends AbstractObject. So, if you're looking for a method on
+    the native tf.keras API but it's not listed here, you might try
+    checking AbstractObject.
+    """
+
+    def send(
```

Comment:
let's open an issue to standardize this & `KerasModel.send` (and potentially any other objects from PySyft main we'd like to handle the same way)

Reply:
looking further into this PR, it looked like `KerasObject` might be the right place to abstract the `send` method

Reply:
as discussed in Slack, I am not able to call send from `KerasObject` in KerasModel` and `KerasLayer`. We can revisit later. thanks